src/main/routes/watch-mk.ts

```diff
@@ -0,0 +1,83 @@
+import { PreClient } from '../services/pre-api/pre-client';
+import { SessionUser } from '../services/session-user/session-user';
+
+import { Application } from 'express';
+import { requiresAuth } from 'express-openid-connect';
+import { v4 as uuid } from 'uuid';
+
+function validateId(id: string): boolean {
+  return /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/.test(id);
+}
+
```

Comment:
We need a method in here which both endpoints can use to check if we're in production or not so we don't have this page accessible in prod for now
note that `NODE_ENV` shouldn't be used for this as that is 'production' for all but local so will need a new value in helm charts - I suggest adding it as a default to `true` in values.yaml then overriding to `false` in stg.yaml, dev.yaml and here: https://github.com/hmcts/sds-flux-config/blob/master/apps/pre/pre-portal/stg.yaml to be false

Reply:
Opened PR here: https://github.com/hmcts/sds-flux-config/pull/5026